cmd/modern/root/uninstall.go

```diff
@@ -116,10 +116,7 @@ func (c *Uninstall) run() {
 			output.Infof(localizer.Sprintf("Removing context %s", config.CurrentContextName()))
 
 			if controller.ContainerExists(id) {
-				output.Infof(
-					"Stopping %s",
-					endpoint.ContainerDetails.Image,
-				)
+				output.Infof(localizer.Sprintf("Stopping %s", endpoint.ContainerDetails.Image))
```

Comment:
>of [](http://example.com/codeflow?start=14&length=2)

what happens if endpoint.ContainerDetails.Image contains a combination that also happens to be a Go format specifier? Won't that cause output.Infof to treat it as a format string?

```go
sub := "%d"
m := localizer.Sprintf("something %s", sub)
output.Infof(m)
```

Reply:
Go libraries typically have 2 versions of string functions - one with "f" and one without. Does `output` have `Info` and `Warn` functions ? If we aren't passing a format string we shouldn't be using and `f` function.

Reply:
Yes. It could be an issue if sub contains format specifier. We have added these functions under output package. I'll add the non `f` variants `output.Info` and `output.Warn` and use them.